src/chart/pie/labelLayout.js

```diff
@@ -222,7 +222,8 @@ export default function (seriesModel, r, viewWidth, viewHeight, sum) {
         var font = labelModel.getFont();
 
         var labelRotate = labelModel.get('rotate')
-            ? (dx < 0 ? -midAngle + Math.PI : -midAngle) : 0;
+            ? +labelModel.get('rotate') * (Math.PI / 180)
+            : (dx < 0 ? -midAngle + Math.PI : -midAngle);
```

Comment:
Note that when `rotate` is set to be `true`, it should use the old algorithm to automatically rotate.
So I think a better way is to check if it's number, if so, use the new algorithm, if not, use the old one.

Reply:
Yes, you are right, I did't notice rotate can be boolean